### DISCUSSION TITLE
Only White Screen after Running SafeExamBrowser | Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by dwikalam** [SafeExamBrowser/seb-win-refactoring#559] [Q&A]
At first, I'm so sorry if this is really out of context of Windows. I really don't know where this question is supposed to be asked while I really need the info. Thank you for the consideration.

As in title, what i did:
1. Follow these README.md instructions https://github.com/eqsoft/seb
However, when I tried demo mode instructions, I could not do that because the module was said not found. However, I think it is not necessarily to do

2. After configuring things as in instructions, I do 
`$ source .../seb/browser/bin/linux/64/start.sh`

3. New window opened with an alert as shown below (the pic: https://ibb.co/zFMVRt8)
I don't install antivirus or sort of things btw
> www.safeexambrowser.org uses an invalid security certificate.
> The certificate is not trusted because the issuer certificate has expired.
> (Error code: sec_error_expired_issuer_certificate)

5. I press ok, then the screen only remains white (the pic: https://ibb.co/NFNhD3z)

Here is what is written in terminal after doing the second step: (the pic: https://ibb.co/HgCfsNM)

> (process:29935): GLib-CRITICAL **: 12:22:11.524: g_slice_set_config: assertion 'sys_page_size == 0' failed
> JavaScript strict warning: resource://modules/xullib.jsm, line 158: ReferenceError: assignment to undeclared variable _logfile
> JavaScript strict warning: resource://modules/xullib.jsm, line 159: ReferenceError: assignment to undeclared variable _logpath
> JavaScript strict warning: resource://modules/xullib.jsm, line 500: ReferenceError: assignment to undeclared variable obj
> JavaScript strict warning: resource://modules/xullib.jsm, line 378: ReferenceError: assignment to undeclared variable k
> xullib: xullib started with application seb
> JavaScript strict warning: resource://gre/modules/TelemetryEnvironment.jsm, line 400: ReferenceError: reference to undefined property this._environment._currentEnvironment.addons
> JavaScript strict warning: resource://modules/xullib.jsm, line 765: ReferenceError: reference to undefined property params[k2]
> seb: seb started...

Please, help me what I should do to be able running SEB. Otherwise, is it not possible anymore to run on Ubuntu?

**Comment by dbuechel** [comment]
This is the deprecated, custom browser engine of the legacy SEB version 2.x, which we do not actively develop nor support anymore. You might have some luck posting your issue in the repository of the browser engine you've mentioned above.
